src/App.tsx

```diff
@@ -1,26 +1,105 @@
 /* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID } from './api/todos';
+import * as servisesTodos from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoInput } from './components/TodoInput/TodoInput';
+import { TodoList } from './components/TodoList/TodoList';
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+import { Footer } from './components/Footer/Footer';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filter, setFilter] = useState<Filter>('All');
+  const [tempTodo, setTempTodo] = useState(null);
+  const [loadingIds, setLoadingIds] = useState<number[]>([]);
+
+  useEffect(() => {
+    servisesTodos
+      .getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage('Unable to load todos');
+      });
+  }, []);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  }, [errorMessage]);
+
+  const filteredTodos = useMemo(() => {
+    if (filter === 'Active') {
+      return todos.filter(todo => !todo.completed);
+    }
+
+    if (filter === 'Completed') {
+      return todos.filter(todo => todo.completed);
+    }
+
+    return todos;
+  }, [filter, todos]);
+
+  function handleDeleteTodo(todoId: number) {
```

Comment:
It is important to follow the same way of creating functions everywhere, fix it everywhere
```suggestion
  const handleDeleteTodo = (todoId: number) => {
```